docs/framework-app_dynamics_agent.md

```diff
@@ -17,14 +19,14 @@ When binding AppDynamics using a user-provided service, it must have name or tag
 
 | Name | Description
 | ---- | -----------
```

Comment:
Based on our work against on-prem instances in the past, none of the items listed were required.  There are defaults for `port` and `ssl-enabled`, `account-access-key` and `account-name` are not required for on-prem, and you must only specify a host name.